graphql_compiler/tests/test_helpers.py

```diff
@@ -279,6 +279,10 @@ def generate_schema_graph(graph_client):
 
 def generate_schema(graph_client, class_to_field_type_overrides=None, hidden_classes=None):
     """Generate schema and type equivalence dict from a pyorient client"""
+    if class_to_field_type_overrides is None:
+        class_to_field_type_overrides = dict()
+    if hidden_classes is None:
+        hidden_classes = set()
```

Comment:
This is getting complicated. This function has a default `set()`. The function it calls has a default `{'V'}`. The function calling this one gives `{'V'}` as an argument. Why not make the default `set()` everywhere, and make the user explicitly ignore it?

Another weird side effect of `{'V'}` being default is that if you want to actually hide another class, you can accidentally unhide `{'V'}`.

Reply:
Ok makes sense. I will do this.